### PR TITLE
KHADRI-30112024-1734-PM-SETUP HOME PAGE USING JSP

### PR DESCRIPTION
**Pull request by SalukulaMahendra**

### CODE REVIEW

File: WebContent/top.jsp
Comment (manvithagali52) [inline WebContent/top.jsp:1]: change to store to mart

File: WebContent/WEB-INF/web.xml
Comment (manvithagali52) [inline WebContent/WEB-INF/web.xml:1]: remove here main.html

**Review by manvithagali52 — DISMISSED** [review]

**Review by roohishaik128 — DISMISSED** [review]

**Review by thasmiyas39 — DISMISSED** [review]

File: src.main.java/com/khadri/mart/fruits/dao/FruitsDao.java [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+			ResultSet resultSet = stmt.executeQuery("select * from fruits");
+
+			while (resultSet.next()) {
+				FruitsForm form = new FruitsForm(resultSet.getString(1), resultSet.getInt(2), resultSet.getDouble(3));
+				listOfClothes.add(form);
+			}
+
+		} catch (Exception e) {
+			System.out.println("Exception occured" + e.getMessage());
+		}
```
Comment (sgreddy123) [inline src.main.java/com/khadri/mart/fruits/dao/FruitsDao.java:141]: where did u closed con? u need to add finally block

File: src.main.java/com/khadri/mart/fruits/dao/FruitsDao.java [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+	public int deleteFruits(String name) {
+        int result = 0;
+        try {
+         con = getConnection();
+        PreparedStatement pstmt = con.prepareStatement("delete from fruits where name=?");
+          pstmt.setString(1, name);
+            result = pstmt.executeUpdate();
+        } catch (Exception e) {
+   e.printStackTrace();
+  }
```
Comment (sgreddy123) [inline src.main.java/com/khadri/mart/fruits/dao/FruitsDao.java]: same , try to add finally block

File: src.main.java/com/khadri/mart/fruits/dao/FruitsDao.java [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+			ResultSet resultSet = stmt.executeQuery("select * from Fruits where name='" + Item_name+"'");
+
+			while (resultSet.next()) {
+				FruitsForm form = new FruitsForm(resultSet.getString(1), resultSet.getInt(2), resultSet.getDouble(3));
+				listOfData.add(form);
+			}
+
+		} catch (Exception e) {
+			System.out.println("Exception occured" + e.getMessage());
+		}
```
Comment (sgreddy123) [inline src.main.java/com/khadri/mart/fruits/dao/FruitsDao.java:112]: add finally block

File: src.main.java/com/khadri/mart/fruits/dao/FruitsDao.java [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+			con = getConnection();
+			PreparedStatement pstmt = con.prepareStatement("insert into fruits values(?,?,?)");
+			pstmt.setString(1, form.getItemName());
+			pstmt.setInt(2, form.getItemQty());
+			pstmt.setDouble(3, form.getItemPrice());
+			result = pstmt.executeUpdate();
+
+		} catch (Exception e) {
+			System.out.println("Exception occured" + e.getMessage());
+		}
```
Comment (sgreddy123) [inline src.main.java/com/khadri/mart/fruits/dao/FruitsDao.java:58]: add finally block

File: src.main.java/com/khadri/mart/fruits/add/servlet/FruitsAddServlet.java [hunk 5]
```diff
@@ -0,0 +1,36 @@
+package com.khadri.mart.fruits.add.servlet;
```
Comment (sgreddy123) [inline src.main.java/com/khadri/mart/fruits/add/servlet/FruitsAddServlet.java]: try to write all servlets in one package

**Review by manvithagali52 — DISMISSED** [review]

**Review by sgreddy123 — APPROVED** [review]

**Review by manvithagali52 — APPROVED** [review]

**Pull request merged**